package.json

```diff
@@ -30,7 +30,7 @@
     "prestart:dev": "yarn run clean",
     "start:dev": "NODE_ENV=development tsc-watch --onSuccess \"node dist/server-dev.js\"",
     "copy-templates": "copyfiles -u 1 src/mailtemplates/**/*.mustache dist/",
-    "db-migrate": "db-migrate --migrations-dir ./src/migrations",
+    "db-migrate": "DATABASE_URL=postgres://unleash_user:passord@localhost:5432/unleash db-migrate --migrations-dir ./src/migrations",
```

Comment:
Wouldn't it be better to add a .env file instead of making sure you can't use the db-migrate task if you want to use a different URL?

Reply:
yes indeed!